### ISSUE TITLE
integration with other plugins

### ISSUE THREAD

**Issue by chchch** [albertgasset/dokuwiki-plugin-codemirror#58]
Hi,

Would it be possible to use a named function instead of an anonymous function as a plugin wrapper, so that it can be called by other plugins?

**Comment by albertgasset** [comment]
Yes, it's possible but I don't think this would be enough. You would need at least these functions:

- Open the editor
- Save content to textarea
- Close the editor

The plugin assumes the textarea has id "wiki__text", and the settings button is added to the div with id "size__ctl". It would be useful to add parameters for these, and to allow multiple instances of the editor.
